### PR TITLE
[TR] Increase speed of Travis CI builds

### PR DESCRIPTION
**Pull request by jmleroux**
- For pull requests and master branch, PHP CS fixers are separated and parallelized on several PHP version builds.

- For branches, only diff files are inspected.

Expected time gains : 
- around 20 minutes to build a PR
- around 5 minutes to build a branch

| Q                    | A
| -------------------- | ---
| Bug fix?             |n
| New feature?         |n
| BC breaks?           |n
| CI currently passes? |y
| Tests pass?          |y
| Scenarios pass?      |y
| Checkstyle issues?*  |n
| PMD issues?**        |n
| Changelog updated?   |n
| Fixed tickets        |
| DB schema updated?   |n
| Migration script?    |n
| Doc PR               |n

### CODE REVIEW

**Comment by sumbobyboys** [comment]
GTM

**Comment by solivier** [comment]
GTM

File: features/Context/FeatureContext.php [hunk 1]
```diff
@@ -10,6 +10,8 @@
 use Behat\MinkExtension\Context\MinkContext;
 use Behat\Symfony2Extension\Context\KernelAwareInterface;
 use Doctrine\Common\DataFixtures\Purger\MongoDBPurger;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerInterface;
```
Comment (skeleton) [inline features/Context/FeatureContext.php]: ObjectManager & ContainerInterface not used

**Comment by grena** [comment]
GTM

**Comment by filipsalpe** [comment]
Couldn't hurt to be able to run it locally too where TRAVIS_BRANCH is not defined

File: features/Context/FeatureContext.php (hunk 1 above)
Comment (jmleroux) [inline features/Context/FeatureContext.php]: used in PhpDoc, but i will use FQCN instead.
Thx.

**Pull request merged**